src/stylesheets/reset.less

```diff
@@ -1,28 +0,0 @@
-/* Eric Meyer's Reset CSS v2.0 - http://cssreset.com */
```

Comment:
that's exactly it, normalize.less from bootstrap is just about browser inconsistencies.